DAPM.Backend.Test.EndToEnd/DAPM.ClientApi.Test.EndToEnd/appsettings.json

```diff
@@ -9,5 +9,8 @@
     "username" : "test",
     "password" : "test",
     "clientSecret" : "gqoRuDK6Z9ntS9vBnSDm3QTrKJ9UO4fW"
```

Comment:
We should remove this from the repo, since it's bad practice 👀

Reply:
I mean remove the example secret

Reply:
It is removed in further pull request.